[2025-05-28T10:20:33.226+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_multi_source.load_task scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T10:20:33.236+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_multi_source.load_task scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T10:20:33.238+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T10:20:33.252+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_task> on 2025-05-27 00:00:00+00:00
[2025-05-28T10:20:33.261+0000] {standard_task_runner.py:60} INFO - Started process 294 to run task
[2025-05-28T10:20:33.264+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_multi_source', 'load_task', 'scheduled__2025-05-27T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/etl_multi.py', '--cfg-path', '/tmp/tmpmljpapga']
[2025-05-28T10:20:33.267+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask load_task
[2025-05-28T10:20:33.357+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_multi_source.load_task scheduled__2025-05-27T00:00:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T10:20:33.435+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_multi_source' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-27T00:00:00+00:00'
[2025-05-28T10:20:33.859+0000] {python.py:201} INFO - Done. Returned value was: All data combined and loaded to SQLite successfully. Table counts: {'penduduk': 5, 'pengguna': 10, 'tugas_selesai': 90, 'pengguna_dan_tugas': 90}
[2025-05-28T10:20:33.880+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_multi_source, task_id=load_task, execution_date=20250527T000000, start_date=20250528T102033, end_date=20250528T102033
[2025-05-28T10:20:33.926+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T10:20:33.947+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
